packages/txservice/src/dispatch.ts

```diff
@@ -364,6 +371,19 @@ export class TransactionDispatch extends ChainRpcProvider {
     const result = await this.sendTransaction(transaction);
     // If we end up with an error, it should be thrown here.
     if (result.isErr()) {
+      const error = result.error;
+      if (error.type === TransactionReverted.type && (error as TransactionReverted).reason === TransactionReverted.reasons.InsufficientFunds) {
+        this.logger.error(
+          "ROUTER HAS INSUFFICIENT FUNDS TO SUBMIT TRANSACTION.",
+          requestContext,
+          methodContext,
+          error.toJson(),
```

Comment:
usually better to use `jsonifyError` instead of `.toJson()` so it doesn't fail here in case the error is not an `NxtpError` type